xonsh/environ.py

```diff
@@ -1203,13 +1201,6 @@ def DEFAULT_VARS():
             "``DEPTH_1_BIT``, ``DEPTH_4_BIT``, ``DEPTH_8_BIT``, ``DEPTH_24_BIT`` "
             "colors. Default is an empty string which means that prompt toolkit decide.",
         ),
-        "PTK_STYLE_OVERRIDES": Var(
```

Comment:
This should probably go through a deprecation cycle of some sort

Reply:
Brought it back, and added a deprecation warning if somebody uses it.